pkg/cli/environment.go

```diff
@@ -102,6 +108,18 @@ func envOr(name, def string) string {
 	return def
 }
 
+func envIntOr(name string, def int) int {
+	if name == "" {
+		return def
+	}
+	envVal := envOr(name, "")
```

Comment:
Might be safer to also pass the default value instead of ""

Reply:
done.